### DISCUSSION TITLE
new server, new domain, same clients...

### DISCUSSION THREAD

**Discussion by hs6666** [Ylianst/MeshCentral#5357] [General]
Hello

I have my existing server (container running on proxmox) at domain1.com say

I cloned the container to a new proxmox server/location, booted it up and changed the domain within config.json to domainb.com say and restarted server

I then went to the clients and edited the msh file on the client to point to doaminb.com instead of domaina.com

Strange things is... some clients connect to the new server, but most still stay connected to the old server (after changing the msh file and restarting the agent)

Also, side note.... when i download client to install from the "domainb meshcentral server" it still refers to the old domaina in its filename?

Grateful for any pointers were I'm going wrong please?

Harry

**Comment by si458** [comment]
if i recall its not as simple as changing a domain name on the server and in the agents because stuff is set in the database
i also have an issue where if i change the group in msh file, the server overwrites the msh file with whatever it has set not what u set

**Comment by hs6666** [comment]
hmmm.... sounds like only way would be too just reinstall meshcentral from scratch, and install the new client over the old client on all the linux/windows hosts and recreate all the groups

wonder if there is a way of running the new client whilst logged into the old clients terminal session, but probably fail as the client would diconnect half way during installation whilst it is overwritten :(

**Comment by hs6666** [comment]
FYI ... update, i noticed that agentupdate in console on new server still showed old download taking place, i deleted "codesign-cert-private" key and crt file and restarted server, and agentupdate now showing correct new domain in console :)

so instead of building new server from scratch, going to try copying the signed agents to the old server (overwriting them) and try agentupdate on the clients still connected to the old server ......
